src/router/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { RouteObject } from 'react-router'
+
+import Events from 'components/pages/Events'
+import Home from 'components/pages/Home'
+import Layout from 'components/pages/Layout'
+import Login from 'components/pages/LogIn'
+import Newsletter from 'components/pages/Newsletter'
+// import ProductDetails from 'components/pages/ProductDetails'
+import Products from 'components/pages/Products'
+import SignUp from 'components/pages/SignUp'
```

Comment:
You can create index in `/pages` and export everything from there, then it's import would look like this:

`import {Events, Home, Layout, ...} from 'components/pages'`